R/doctor.R

```diff
@@ -91,7 +93,21 @@ dr_github <- function(path = ".") {
 
   if (identical(Sys.getenv("GITHUB_PAT"), ""))
     msg[["PAT"]] <- paste("* GITHUB_PAT environment variable not set",
-      "(this is not critical unless you want to install private repos)")
+      "(this is not necessary unless you want to install private repos",
+      "or connect local repos to GitHub)")
+
+  pkg <- as.package(path)
+  desc_path <- file.path(pkg$path, "DESCRIPTION")
+  desc <- read_dcf(desc_path)
+  desc <- compact(desc[c("URL", "BugReports")])
```

Comment:
I check for dummy info because `use_github_links()` will dutifully insert placeholder links, even if GitHub remote can't be detected. It felt like good citizenship to build in a check for this weird state. No I cannot imagine anyone doing this on purpose!

Good point re: also checking for empty Url and BugReports. Will add.